wis2box/metadata/station.py

```diff
@@ -60,6 +62,33 @@ def station():
     pass
 
 
+def load_topics():
+    oaf = Features(API_URL)
+
+    dm = oaf.collection_items('discovery-metadata')
+
+    for topic in dm['features']:
+        for link in topic['properties']['associations']:
+            if link['type'] == 'OAFeat':
+                yield link
+
+
+def check_station_topics(topics, wigos_id):
+    oaf = Features(API_URL)
+
+    for topic in topics:
+        try:
+            obs = oaf.collection_items(
+                topic["title"], wigos_station_identifier=wigos_id)
```

Comment:
`topic['id'], wigos_station_identifier=wigos_id)`

Reply:
As discussed, thought you were using the entire collection JSON (vs just the link object).  Fine to keep as is then.  Suggest to rename those funcs though to `load_datasets` and `check_station_datasets`.